sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -10,7 +10,7 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <MinorProductVersion>1</MinorProductVersion>
-    <VersionSuffix>-preview2</VersionSuffix>
+    <VersionSuffix>-preview3</VersionSuffix>
```

Comment:
May update the version based on feedback from the team.

Reply:
Is there an alternative version you are considering? If this is to remain in preview, then the change you made looks good to me.

Reply:
Yea, I was discussing with the folks whether we should move this package out of preview (since the SDK GA package relies on this). But that is not a blocker for checking in this PR at the moment. So going to check in this version for now.